startup/00-nsls2-tools.py

```diff
@@ -56,5 +57,8 @@ def wait_for_connection(self, timeout=DEFAULT_CONNECTION_TIMEOUT):
 runengine_metadata_dir = os.path.expanduser("/nsls2/data/csx/shared/config/RE-metadata")
 RE.md = PersistentDict(runengine_metadata_dir)
 
+bec = BECwithTicks()
```

Comment:
I don't understand how this is supposed to then work with [this use of `bec`](https://github.com/NSLS-II-CSX/profile_collection/blob/429cf3846eb42aecddfe5ad839c007a1b8f79f58/startup/99-helper_functions.py#L7) that is used after this file is executed.

Reply:
Because the change inherits from `BestEffortCallback`, this should maintain all of the downstream functionality as expected.

Reply:
Bingo!